index.html

```diff
@@ -156,12 +156,12 @@ <h1 id="manifesto" class="title text-center">Our <span>Manifesto</span></h1>
 -->
 	      <p class="lead">Web apps today have incredible power</p>
 	      <p>
-	      When we use most modern web applications,
-	      the majority of computation and storage over your data happens
-	      on servers owned and operated by the developer of the website and their partners.
+	      When we use modern web applications,
+	      the majority of computation and storage of our data happens
+	      on servers owned and operated by big companies.

```

Comment:
The common case in our framework is still to use IaaS Cloud providers (e.g. EC2), so that's still operated by big companies. How can we distinguish the difference between this and the web dev controlling your stuff?